src/NewProject/ProjectManager/project_manager.cpp

```diff
@@ -1359,14 +1274,8 @@ int ProjectManager::deleteFileSet(const QString& strSetName) {
   int ret = 0;
   ProjectFileSet* proFileSet =
       Project::Instance()->getProjectFileset(strSetName);
-  if (proFileSet && PROJECT_FILE_TYPE_DS == proFileSet->getSetType() &&
```

Comment:
Are these entries no longer needed?

Reply:
yes, the logic was changed long time ago and this doesn't make sense anymore